### ISSUE TITLE
TypeError: Retry.__init__() got an unexpected keyword argument 'method_whitelist'

### ISSUE THREAD

**Issue by Tarkhubal** [iamkubi/pydactyl#76]
**Describe the bug**
I have an error and I think it's because of the new version, I had never seen it before

I had the error and my program analyzed by BingAI and it told me this:
It seems that this library uses urllib323 to make HTTP requests. As I said before, the method_whitelist argument was renamed to allowed_methods in urllib32 version 1.26.0.

**To Reproduce**
The error :
```ps
File "/home/container/.local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 841, in _do_call
     return await self._callback(self.binding, interaction, **params) # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^
   File "/home/container/cogs/manage_server.py", line 111, in servers_stats
     api = PterodactylClient(guild_data["url"], guild_data["PteroAPIKey"])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^
   File "/home/container/.local/lib/python3.11/site-packages/pydactyl/api_client.py", line 69, in __init__
     adapter = http_adapter(backoff_factor=backoff_factor,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/home/container/.local/lib/python3.11/site-packages/pydactyl/api_client.py", line 16, in http_adapter
     retries = requests.packages.urllib3.util.retry.Retry(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Retry.__init__() got an unexpected keyword argument 'method_whitelist'
```

**Environment**
 - py-dactyl version 2.0.1
 - Python version 3.11
 - discord.py version 2.2.3
 - requests version 2.30.3

**Additional context**
I'm sure it's not my program, it worked perfectly before the new version
Either it's the requests module or it's the py-dactyl module

**Comment by Tarkhubal** [comment]
Okay it's the request module
I went back a version back and that's it

**Comment by Tarkhubal** [comment]
I found the same issues in the requests module github

**Comment by Tarkhubal** [comment]
I found the same issues in the requests module github

**Comment by iamkubi** [comment]
Fixed in the 2.0.2 release
